From e38c8f8c6427ecf52c2e1aea56588af52a682fbb Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Fri, 15 Oct 2021 05:48:14 -0700
Subject: [PATCH] python-3.9 compatibility

tp_print was removed from python-3.9:
https://bugs.python.org/issue36974

but wasn't used since 3.0:
https://bugs.python.org/issue39361

so just remove it without ifdefs on PY_MINOR_VERSION

Upstream-Status: Pending

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 _proj.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/_proj.c b/_proj.c
index 9de34a3..e3c7423 100644
--- a/_proj.c
+++ b/_proj.c
@@ -8599,14 +8599,12 @@ static int __Pyx_modinit_type_init_code(void) {
   __Pyx_RefNannySetupContext("__Pyx_modinit_type_init_code", 0);
   /*--- Type init code ---*/
   if (PyType_Ready(&__pyx_type_5_proj_Proj) < 0) __PYX_ERR(0, 111, __pyx_L1_error)
-  __pyx_type_5_proj_Proj.tp_print = 0;
   if ((CYTHON_USE_TYPE_SLOTS && CYTHON_USE_PYTYPE_LOOKUP) && likely(!__pyx_type_5_proj_Proj.tp_dictoffset && __pyx_type_5_proj_Proj.tp_getattro == PyObject_GenericGetAttr)) {
     __pyx_type_5_proj_Proj.tp_getattro = __Pyx_PyObject_GenericGetAttr;
   }
   if (PyObject_SetAttr(__pyx_m, __pyx_n_s_Proj, (PyObject *)&__pyx_type_5_proj_Proj) < 0) __PYX_ERR(0, 111, __pyx_L1_error)
   __pyx_ptype_5_proj_Proj = &__pyx_type_5_proj_Proj;
   if (PyType_Ready(&__pyx_type_5_proj_Geod) < 0) __PYX_ERR(0, 512, __pyx_L1_error)
-  __pyx_type_5_proj_Geod.tp_print = 0;
   if ((CYTHON_USE_TYPE_SLOTS && CYTHON_USE_PYTYPE_LOOKUP) && likely(!__pyx_type_5_proj_Geod.tp_dictoffset && __pyx_type_5_proj_Geod.tp_getattro == PyObject_GenericGetAttr)) {
     __pyx_type_5_proj_Geod.tp_getattro = __Pyx_PyObject_GenericGetAttr;
   }
